hmda-data-publisher/src/main/scala/hmda/publisher/helper/QuarterTimeBarrier.scala

```diff
@@ -52,10 +52,10 @@ object QuarterTimeBarrier {
       case Period.y2020Q3 => LocalDate.ofYearDay(2020,rulesConfig.qf.q3.endDayOfYear)
       case Period.y2021Q1 => LocalDate.ofYearDay(2021,rulesConfig.qf.q1.endDayOfYear)
       case Period.y2021Q2 => LocalDate.ofYearDay(2021,rulesConfig.qf.q2.endDayOfYear)
-      case Period.y2021Q3 => LocalDate.ofYearDay(2021,rulesConfig.qf.q3.endDayOfYear)
+      case Period.y2021Q3 => LocalDate.ofYearDay(2021,rulesConfig.qf.q3.endDayOfYear - 1)
```

Comment:
Did you get unit tests to pass with this change? I see this failure when I run unit tests with your change: 
```
Protected code shouldn't run but it did. Date: 2021-09-22, period: y2021Q1
ScalaTestFailureLocation: hmda.publisher.helper.QuarterTimeBarrierTest at (QuarterTimeBarrierTest.scala:50)
org.scalatest.exceptions.TestFailedException: Protected code shouldn't run but it did. Date: 2021-09-22, period: y2021Q1
	at org.scalatest.Assertions.newAssertionFailedException(Assertions.scala:530)
	at org.scalatest.Assertions.newAssertionFailedException$(Assertions.scala:529)
	at org.scalatest.FreeSpec.newAssertionFailedException(FreeSpec.scala:1739)
	at org.scalatest.Assertions.fail(Assertions.scala:1091)
	at org.scalatest.Assertions.fail$(Assertions.scala:1087)
	at org.scalatest.FreeSpec.fail(FreeSpec.scala:1739)
	at hmda.publisher.helper.QuarterTimeBarrierTest.test(QuarterTimeBarrierTest.scala:50)
	at hmda.publisher.helper.QuarterTimeBarrierTest.testQuarter(QuarterTimeBarrierTest.scala:35)
	at hmda.publisher.helper.QuarterTimeBarrierTest.$anonfun$new$1(QuarterTimeBarrierTest.scala:17)
```

Reply:
Hmm, this test passes on my end, I'll check on this

Reply:
Strange, this is might be an IntelliJ issue, since the tests pass for me in sbt